src/stories/Foundation/Table/TableHeaderItem.tsx

```diff
@@ -61,7 +62,7 @@ const TableHeaderItem = ({
         </Flex>
       </TH>
       {isSortable && align === 'right' && (
-        <TH align='right' width='30px'>
+        <TH align='right' width={rem(30)}>
```

Comment:
Maybe double check this one since it is a prop?

Reply:
Will do, but it shouuuuuld work :)))